server/auth/assets/templates/profile.html.tpl

```diff
@@ -18,6 +20,66 @@
 		</div>
 		{{ end }}
 
+        {{ if .avatarEnabled }}
+        <div class="mb-3">
+            <label for="profileFormHandle">{{ tr "profile.template.form.avatar.label" }}</label>
+            <div class="d-block">
+                <div class="d-flex justify-content-between">
+                    <div>
+                        <img style="height: 4rem; width: 4rem;" class="rounded-circle" src="{{ .form.avatarUrl }}" alt="Profile Photo">
+                    </div>
+                    <div>
+                        <button
+                            name="avatar-delete"
+                            value="avatar-delete"
+                            class="btn btn-danger"
+                        >
+                        {{ tr "profile.template.form.avatar.delete" }}
+                        </button>
+
+                        <label for="avatar" class="p-2 ml-3 bg-light text-dark rounded">
+                        {{ tr "profile.template.form.avatar.update" }}
+                        </label>
+                        <input id="avatar" name="avatar" type="file" class="sr-only" accept="image/*">
+                    </div>
+                </div>
+            </div>
+        </div>
+        {{ end }}
+
+
+        {{ if .initialsEnabled }}
+        <div class="mb-3">
+            <label for="avatarInitials">{{ tr "profile.template.form.avatar-initial.label" }}</label>
+            <div class="d-block">
+                <div class="d-flex justify-content-start align-items-center">
+                    <div>
+                        <input class="col-10 form-control" type="text" id="avatarInitials" name="avatar-initials" value="{{ .form.avatarInitial }}">
```

Comment:
Should we take the initials from the user's name instead?

Reply:
Yes, I believe we can do this:

1.  When the name and handle don't exist, we can use the first letter of the user's email address for example `name@tld.dev` would be `N`.
2. When the handle exists and the name doesn't we can use the first letter of the handle for example `testuser` would be `T`.
3. When the name exists, we can use the first two letters if the user provided only the first name for example 'Francis' would be `FR`, or the first characters for the first and last name when the user provides their fullname, for example `Francis Mumbi` would be `FM`.